src/sentry/relocation/tasks/transfer.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+from celery import Task
+from django.db.models import Subquery
+from django.utils import timezone
+
+from sentry.relocation.models.relocationtransfer import (
+    MAX_AGE,
+    RETRY_BACKOFF,
+    BaseRelocationTransfer,
+    ControlRelocationTransfer,
+    RegionRelocationTransfer,
+)
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger("sentry.relocation.tasks")
+
+
+@instrumented_task(
+    name="sentry.relocation.transfer.find_relocation_transfer_control",
+    queue="relocation.control",
+    silo_mode=SiloMode.CONTROL,
+)
+def find_relocation_transfer_control() -> None:
+    _find_relocation_transfer(ControlRelocationTransfer, process_relocation_transfer_control)
+
+
+@instrumented_task(
+    name="sentry.relocation.transfer.find_relocation_transfer_region",
+    queue="relocation",
+    silo_mode=SiloMode.REGION,
+)
+def find_relocation_transfer_region() -> None:
+    _find_relocation_transfer(RegionRelocationTransfer, process_relocation_transfer_region)
+
```

Comment:
Do we want any metrics to track task progress & failure/success rate? Might be too early to add since this looks like a dry run.

Reply:
I'll add logs to the delivery tasks, with how many relocations we process I don't think metrics will be that useful.